middleware/queue/source/manager/handle.cpp

```diff
@@ -472,7 +474,13 @@ namespace casual
                         for( auto& name : message.content.queues)
                         {
                            if( auto queue = get_queue( name))
-                              reply.content.queues.emplace_back( std::move( name), queue->order);
+                           {
+                              auto& value = reply.content.queues.emplace_back();
+                              value.name = name;
```

Comment:
Use `value.name = queue.name` and/or `std.move( name)` or preferable a transform-method